src/Bicep.RegistryModuleTool/Program.cs

```diff
@@ -44,10 +53,7 @@ private static Parser CreateParser()
         }
 
         private static void ConfigureHost(IHostBuilder builder) => builder
-            .ConfigureServices(services => services
-                .AddSingleton<IEnvironmentProxy, EnvironmentProxy>()
```

Comment:
>AddSingleton [](http://example.com/codeflow?start=17&length=12)

Are there any package dependencies we can remove from the BRM tool project?

Reply:
Yes, found a package that could have been removed even before this PR 🤦‍♂️.

Reply:
Nice!